robots.json

```
@@ -90,6 +90,13 @@
         "frequency": "Unclear at this time.",
         "description": "cohere-training-data-crawler is a web crawler operated by Cohere to download training data for its LLMs (Large Language Models) that power its enterprise AI products. More info can be found at https://darkvisitors.com/agents/agents/cohere-training-data-crawler"
     },
+    "Crawlspace": {
+        "operator": "[Crawlspace](https://crawlspace.dev)",
+        "respect": "[Yes](https://news.ycombinator.com/item?id=42756654)",
+        "function": "Scrapes data",
+        "frequency": "Unclear at this time.",
+        "description": "Provides crawling services for any purpose, but most likely to be used for AI model training."
```

I think a slightly weaker statement might be more accurate:
```suggestion
        "description": "Provides crawling services for any purpose, probably including AI model training."
```